packages/material-ui-unstyled/src/SliderUnstyled/SliderValueLabelUnstyled.js

```diff
@@ -23,14 +23,10 @@ const useValueLabelClasses = (props) => {
 /**
  * @ignore - internal component.
  */
-function ValueLabel(props) {
-  const { children, className, value, valueLabelDisplay, components = {}, theme } = props;
+function SliderValueLabelUnstyled(props) {
+  const { children, className, value, components = {}, theme } = props;
   const classes = useValueLabelClasses(props);
 
-  if (valueLabelDisplay === 'off') {
```

Comment:
This is now reverted to the original logic, wasn't caught previously as we didn't have screen comparison with the base Slider.

Reply:
Should this change be moved into a different pull request?